src/window_base.cpp

```diff
@@ -49,6 +65,26 @@ void Window_Base::Update() {
 		SetWindowskin(Cache::System(windowskin_name));
 	}
 	SetStretch(Game_System::GetMessageStretch() == RPG::System::Stretch_stretch);
+	UpdateMovement();
+}
+
+void Window_Base::UpdateMovement() {
+	if (!IsMovementActive()) {
+		return;
+	}
+	current_frame++;
+	if (!IsMovementActive()) {
+		SetX(new_position[0]);
+		SetY(new_position[1]);
+	}
+}
+
+void Window_Base::Draw() {
+	if (IsMovementActive()) {
+		SetX(old_position[0] + (new_position[0] - old_position[0]) * current_frame / total_frames);
+		SetY(old_position[1] + (new_position[1] - old_position[1]) * current_frame / total_frames);
+	}
```

Comment:
> 2.3: RM2000 + RM2003: The battle command window (both left [manual, 

Logic updates can't be done in Draw(), they will be skipped when frame skipping / lag happens or when Fast Forward is used

Reply:
I moved this logic to `UpdateMovement()`. Seems to work fine so not sure why this was added to `Draw()` before.